doc/main/reference/task_group_extensions.rst

```diff
@@ -46,22 +46,27 @@ Synopsis
                task_handle defer(F&& f);
                    
                void run(task_handle&& h);
+               
+               task_group_status run_and_wait(task_handle&&);
 
-               //only F return type requirements have changed              
+               //only the requirements for the type of return value F have changed              
```

Comment:
`F` is the function provided by user. That function might have a return value (or might not). Requirement changes mentioned in this line are about type of return value of function `F`.

Reply:
I see, then maybe something like that:
``only the requirements for the type of return value of function F changed``

Reply:
I would change it to be `only the requirements for the return type of function F are changed`

Reply:
Sounds great